### ISSUE TITLE
Accessibility: Text area: Label “10 characters remaining” is not read when text area is accessed

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3439]
Description: Label “10 characters remaining” is not read when text area is accessed

Expected: Ideally, corresponding label should be read when UI is accessed. 

Attached screenshot: 
![image](https://user-images.githubusercontent.com/32538291/94518032-c02be300-0246-11eb-8f51-03934b15ec98.png)
